css/easyzoom.css

```diff
@@ -46,6 +46,22 @@
 	background: #FFF;
 }
 
+.easyzoom-lens {
+	position: absolute;
+	top: 0;
+	left: 0;
+	background-color: #FFF;
+	background-color: rgba(255, 255, 255, 0.35);
+	border: 1px solid #CCC;
+	border-color: rgba(0, 0, 0, 0.35);
+	width: auto;
+	height: auto;
+	cursor: pointer;
```

Comment:
I actually had been pondering that, however, I ended up needing to be able to click through the lens so as to open a modal. I wasn't sure whether the cursor should be pointer, remain crosshair, or be invisible when using the lens overlay. Maybe that particular feature shouldn't be included here. Thoughts?